go.mod

```diff
@@ -30,7 +30,7 @@ require (
 	github.com/go-telegram-bot-api/telegram-bot-api v4.6.4+incompatible
 	github.com/go-webauthn/webauthn v0.6.0
 	github.com/golang-jwt/jwt/v4 v4.5.0
-	github.com/google/uuid v1.4.0
+	github.com/google/uuid v1.6.0
```

Comment:
imho, it is unnecessary to change go.mod , it will be better to open a new pr to upgrade packages

Reply:
@Bsheepcoder not next time. It needs to be fixed "this time", or this PR will not pass